templates/flashes_helper.rb

```diff
@@ -0,0 +1,5 @@
+module FlashesHelper
+  def user_facing_flashes
+    flash.to_hash.slice("alert", "error", "notice", "success")
```

Comment:
Is there a way to read from this api instead? http://blog.remarkablelabs.com/2012/12/register-your-own-flash-types-rails-4-countdown-to-2013

Seems like that'd be likely to be in line with what we want and more dry (register a flash type and you are all set). As long as you didn't register a type for the other uses of flash... Which I think is a fair tradeoff.